test/feederer_test.exs

```diff
@@ -47,8 +47,8 @@ defmodule FeedererTest do
 
 
     modified = parsed[:headers]
-                      |> Enum.find(fn(x) -> elem(x, 0) == "last-modified" end)
-                      |> elem(1)
+               |> Enum.find(fn(x) -> elem(x, 0) == "last-modified" end)
+               |> elem(1)
```

Comment:
Not important, but I like to indent this kind of "pipe story" like that:

    modified =
      parsed[:headers]
      |> Enum.find(fn(x) -> elem(x, 0) == "last-modified" end)
      |> elem(1)

It makes it easier to avoid long lines and the indentation level does not depend on the length of the variable.